compose/src/main/kotlin/io/github/kakaocup/compose/utilities/Extensions.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.kakaocup.compose.utilities
+
+import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
+import io.github.kakaocup.compose.KakaoCompose
+import io.github.kakaocup.compose.exception.KakaoComposeException
+
+fun SemanticsNodeInteractionsProvider?.orGlobal(): SemanticsNodeInteractionsProvider =
+    this
+        ?: KakaoCompose.Global.semanticsProvider
+        ?: throw KakaoComposeException("SemanticsProvider not is null: Provide via constructor or use KakaoComposeTestRule")
```

Comment:
"SemanticsProvider should not be null: ..."?

Reply:
Typo fixed